src/components/actions/index.js

```diff
@@ -0,0 +1,68 @@
+const postsRequested = () => {
+  return {
+    type: 'FETCH_POSTS_REQUEST'
```

Comment:
Create constants for this strings. They can be used in other reducers lately.

Also, pls move reducer and initial state of this domain n this file - you will receive module. I told about them on the lecture